### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by m26unkwn**
Feature
1. Filter
2. Products List

### CODE REVIEW

File: styles/home.css [hunk 1]
```diff
@@ -186,8 +182,13 @@
   .logotext {
     display: none;
   }
-
+  .link-btn {
+    display: none;
+  }
+  .search-bar {
```
Comment (j24m) [inline styles/home.css:191]: inconsistent indentation, please use beatify extension.

File: styles/home.css [hunk 2]
```diff
@@ -186,8 +182,13 @@
...
+    width: 100%;
+  }
   .search-bar > input {
-    width: 20%;
```
Comment (j24m) [inline styles/home.css:191]: here too.

File: styles/product.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  display: grid;
+  grid-template-columns: 15rem 1fr;
+  margin-top: 3rem;
+}
+
+.filter {
+  padding: 0 1rem;
+  color: var(--heading-text-color);
+}
+.filter-head {
```
Comment (j24m) [inline styles/product.css:25]: provide space here too

**Review by j24m — APPROVED** [review]
Hey, you can make the changes in the code I suggested.
Now coming to your UI page :-
1. Nav-bar : You can align your logo at the left end and rest of the elements at the right end.
2. Badges on icon in nav bar : You can position them in a way that they are touching each other's edge.
3. Card : wishlist icon on card has white background which is not looking really good, you can make it a bit grayish, but ofcourse your choice :) 
4. Finally, I appreciate the work you have put in, great job.

File: screens/product.html [hunk 4]
```diff
@@ -0,0 +1,1189 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (PuneetP16) [inline screens/product.html:2]: Add pointer on cursor hover

image

**Review by PuneetP16 — APPROVED** [review]
Code looks good to merge.

File: styles/product.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  margin-bottom: 1rem;
+}
+
+.filter-section > * {
+  margin: 1rem 0;
+  list-style-type: none;
+}
+
+.filter-section > li > * {
```
Comment (ashishkatiyar2404) [inline styles/product.css:34]: for now, it's okay. buy a small suggestion use classes for styling it will maintain your Specificity.

**Review by ashishkatiyar2404 — APPROVED** [review]
both, your code and UI look good to me.
you can merge💯

File: styles/home.css (hunk 1 above)
Comment (m26unkwn) [reply styles/home.css:191]: Thanks For valuable Review @j24m  will resolve these issues.

File: screens/product.html (hunk 4 above)
Comment (m26unkwn) [reply screens/product.html:2]: Thanks @PuneetP16  Just added the cursor pointer .

File: styles/product.css (hunk 5 above)
Comment (m26unkwn) [reply styles/product.css:34]: Sure @ashishkatiyar2404 I will do use the classes here.

Reason Why I used css like this here.

1. I did not give any classes to list so thats why i am doing like this.
2.  I wanted every list item to look same so its like shorthand.
3. Rather adding classes to every list i prefer this method. 


But thanks for now on i will look over this piece of advice.

**Pull request merged**